src/bundle/ui-dev/src/modules/common/thumbnail/thumbnail.tsx

```diff
@@ -1,9 +1,15 @@
-import React from 'react';
-import PropTypes from 'prop-types';
-
 import Icon from '../icon/icon';
 
-const Thumbnail = ({ thumbnailData, iconExtraClasses, contentTypeIconPath }) => {
+interface ThumbnailProps {
+    thumbnailData: {
+        mimeType: string;
+        resource: string;
+    };
+    iconExtraClasses?: string;
+    contentTypeIconPath?: string | null;
```

Comment:
What about adjusting the code so that we can resign from using _null_ in the type? 🤔 
```suggestion
    contentTypeIconPath?: string;
```